SonarQube.Bootstrapper/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ERROR_VersionMismatch" xml:space="preserve">
     <value>The C# plugin installed on the server is not compatible with the SonarQube.MSBuild.Runner.exe - either check the compatibility matrix or get the latest versions for both. </value>
   </data>
+  <data name="INFO_CopiedTargets" xml:space="preserve">
+    <value>Copied {0} to {1}</value>
```

Comment:
This is accurate but low-level and doesn't give the user much idea which the copy is being done. A higher-level and possibly more helpful message would be "Installed SonarQube.MSBuild.Integration.ImportBefore.targets to [xxx]".

Alternatively, you could write an initial header message to give the overall context e.g.
"Installing the SonarQube integration loader targets file:
* installed to [X]
* targets at [Y] are up-to-date
* overwriting targets at [Z] (different version)"